net/k6/Portfile

```diff
@@ -30,3 +32,5 @@ destroot {
     xinstall -m 0644 -W ${worksrcpath} README.md LICENSE.md \
         ${destroot}${docdir}
 }
+
+github.tarball_from archive
```

Comment:
Small nitpick: move this above checksums.

Reply:
Done